tests/lit-tests/1755.ispc

```diff
@@ -0,0 +1,6 @@
+//; RUN: %{ispc} %s -o %t.o --nowrap --target=avx2-i32x8 2>&1
```

Comment:
Either, a restriction for running only on X86 is missing or you need to have ``--target=host`` and ifdef chan to cover compilation width for 4/8/16.